indico/http/serialization.py

```diff
@@ -85,5 +85,7 @@ def zip_serialization(content, charset=None):
         "image/png": image_serialization,
         "image/jpg": image_serialization,
         "application/zip": zip_serialization,
+        "application/x-gzip": raw_bytes,
```

Comment:
This makes sense - but makes me wonder what the gzip option on `deserialize` is for... @goatrocks ? Any idea where we'd use that?

Reply:
it's gets automatically set if the file you're downloading ends in '.gz'. Tbh not great cause now we've kinda created an incorrect that stuff will magically work

Reply:
occurred to me, do we want to keep doing the `.gz` reading? or should we only decompress if the mime type is correct?

Reply:
I would say refer to mime-type and not to file name - but do we ever... set bad mimetypes? not sure... so to be safe, could just `if endswith(.gz) or mimetype == gzip`
that's just my guess tho

Reply:
nginx sets the mime type, so i doubt we ever send bad ones. we'd have to manually override the response header which i highly doubt we do anywhere

Reply:
What @thearchitector said is right, it gets set in `client.py` at line 131-138, and assumes that gzip content will end in .gz, but gzip content tends to come with the mime type set not the file extension so if it's enabled on a server this code doesn't work (hence the incorrect assumption)

I would lean towards removing the .gz assumption, but I'm also wary of removing things people may or may not be relying on, but then again I can't think of a situation in which we'd be returning a .gz file..